downloadAndInsertOpenCV.sh

```diff
@@ -2,24 +2,24 @@
 
 # ios
 version=4.1.0
```

Comment:
why don't you just change this version?

and why do you prefer an older version?